src/presentation/http/router/user.test.ts

```diff
@@ -40,4 +40,36 @@ describe('User API', () => {
       expect(body.message).toBe('You must be authenticated to access this resource');
     });
   });
-});
\ No newline at end of file
+  describe('POST /user/editor-tools', () => {
```

Comment:
I think, testcases for tools logic should be in separate file

Reply:
replaced